rpm/generic/zfs.spec.in

```diff
@@ -214,6 +214,8 @@ Summary:        Dracut module
 Group:          System Environment/Kernel
 Requires:       %{name}%{?_isa} = %{version}-%{release}
 Requires:       dracut
+Requires:       gawk
```

Comment:
Can we do just `awk` here to accommodate BusyBox awk?  Or do we use GNU extensions?

Reply:
Initially I tried using `awk` but ran in to problems since the CentOS/Fedora repositories don't include a package by that name, or an alias.   So I was forced to switch to `gawk`.  Although any of the other `awk` variants should work fine.  If someone knows a better generic way to specific this I'm all for it.  Maybe `Requires: /usr/bin/awk`?